modules/configuring-an-authentication-provider-in-central.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * cli/getting-started-cli.adoc
+:_content-type: PROCEDURE
+[id="configuring-an-authentication-provider-in-central_{context}"]
+= Configuring an authentication provider in Central
+
+[role="_abstract"]
+You can configure an authentication provider in Central and initiate the login process with the `roxctl` CLI.
+Set the variable `ROX _ENDPOINT`, initiate the login process with the `roxctl central login` command, select the authentication provider in a browser window, and retrieve the token information from the `roxctl` CLI.
+
+.Procedure
+
+. Select an authentication provider of your choice, such as OpenID Connect (OIDC) with fragment or query mode, and configure it in Central.
+
+. Run the following command to set the `ROX _ENDPOINT` variable to Central hostname and port:
++
+[source,terminal,subs="+quotes"]
+----
+export ROX_ENDPOINT=_<central_hostname:port>_
+----
+
+. Run the following command to initiate the login process to Central:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl central login
+----
+
+. Within `roxctl` CLI, a URL is printed as output and you are redirected to a browser window where you can select the authentication provider you want to use.
+
+. Once you have successfully logged in with the authentication provider, the browser window indicates that authentication was successful and you can close the browser window.
+
+. `roxctl` CLI prints your token information. This includes details such as the access token, the expiration time of the access token, the refresh token if one has been issued, and notification that these values are stored locally.
```

Comment:
s/`roxctl` CLI/The `roxctl` CLI

s/prints/displays (I know it prints-to-screen. But I think _displays_ is clearer.)

Maybe combine the sentences? 

> token information including details....